spec/feature/posts_index_spec.rb

```diff
@@ -47,14 +38,4 @@
   it 'Pagination' do
     expect(page).to have_button('Pagination')
   end
-
```

Comment:
- [ ] Great job working on this milestone. You are currently missing the test that `redirects to the post's show page when you click on a post.`